cmd/oras/internal/option/format.go

```diff
@@ -74,16 +78,16 @@ type Format struct {
 	AllowedTypes []*FormatType
 }
 
-// ApplyFlag implements FlagProvider.ApplyFlag.
+// ApplyFlags implements FlagProvider.ApplyFlag.
 func (opts *Format) ApplyFlags(fs *pflag.FlagSet) {
 	buf := bytes.NewBufferString("[Experimental] Format output using a custom template:")
 	w := tabwriter.NewWriter(buf, 0, 0, 2, ' ', 0)
 	for _, t := range opts.AllowedTypes {
 		_, _ = fmt.Fprintf(w, "\n'%s':\t%s", t.Name, t.Usage)
 	}
-	w.Flush()
+	_ = w.Flush()
 	// apply flags
-	fs.StringVar(&opts.FormatFlag, "format", opts.FormatFlag, buf.String())
+	fs.StringVar(&opts.FormatFlag, "format", FormatTypeText.Name, buf.String())
```

Comment:
This is wrong, some command doesn't have text output, e.g. `oras discover`. Should only set the default value to `FormatTypeText.Name` only when `opts.FormatFlag` is not changed.